### PR TITLE
Update playground UI based on user request

### PR DESCRIPTION
**Pull request by gelisam**
From Google Jules:

1. Add 'safe in practice'/'unsafe in practice' indicator:
   - Displays below the heatmap, indicating if any critical 'emergency stop'
     patterns are misclassified as 'incinerate'.
   - Dynamically updates with network training.

2. Remove Regularization dropdowns:
   - Removed 'Regularization' and 'Regularization rate' controls from the UI.
   - Associated logic and state properties have been removed. Network now
     operates without L1/L2 regularization.

3. Modify Learning Rate dropdown highlighting:
   - When the learning rate is automatically adjusted due to zigzagging,
     only the dropdown input field itself will flash yellow.

4. Remove 'Test loss' display and line chart series:
   - Removed 'Test loss' text from the output metrics.
   - The line chart now only displays 'Training loss'.
   - Underlying chart logic updated to handle a single data series.

### CODE REVIEW

**Comment by gelisam** [comment]
The CI action didn't run. Maybe if I close this PR and open new one?
